apidoc/v1/index.html

```diff
@@ -3689,4 +3689,4 @@ <h3 id='possible-exceptions-3'>Possible exceptions</h3>
       </div>
     </div>
   </body>
-</html>
+</html>
```

Comment:
@salmoneatenbybear This file is still in the diff. 

You can 

`git checkout master apidoc/v1/index.html` to reset